Tests/RxCocoaTests/UITextField+RxTests.swift

```diff
@@ -13,35 +13,52 @@ import XCTest
 
 // UITextField
 final class UITextFieldTests : RxTest {
+    
     func test_TextCompletesOnDealloc() {
         ensurePropertyDeallocated({ UITextField() }, "a", comparer: { $0 == $1 }) { (view: UITextField) in view.rx.text }
     }
     func test_ValueCompletesOnDealloc() {
         ensurePropertyDeallocated({ UITextField() }, "a", comparer: { $0 == $1 }) { (view: UITextField) in view.rx.value }
     }
-
+    
     func testSettingTextDoesntClearMarkedText() {
         let textField = UITextFieldSubclass(frame: CGRect.zero)
-
         textField.text = "Text1"
-        textField.set = false
+        textField.isSettedText = false
         textField.rx.text.on(.next("Text1"))
-        XCTAssertTrue(!textField.set)
+        XCTAssertTrue(!textField.isSettedText)
         textField.rx.text.on(.next("Text2"))
-        XCTAssertTrue(textField.set)
+        XCTAssertTrue(textField.isSettedText)
+    }
+    
+    func testLabel_attributedTextObserver() {
```

Comment:
Test title says `testLabel`. It should probably be just `test` because there is no label that we are testing inside this test.